### ISSUE TITLE
[Feature Request] v-data-table: footer not WCAG compliant

### ISSUE THREAD

**Issue by jwrr07** [vuetifyjs/vuetify#17127]
### Problem to solve
The pagination drop down should be WCAG compliant.

With the chrome plugin "Accessibility Insights for Web" 
( https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni )
you can see, that the drop down is "nested interactive". 

As far as i researched it, the problem wasnt there with Vuetify version 1.5.

To see this error, you can use the plugin and build a simple datatable, for example:
https://play.vuetifyjs.com/#eNrNVd1u0zAUfhUvNwVpblZNQlB11bjjCiFuuFinyo1PVoNjW7aTbUx9d85xkq5pO1aBhKjUJOf/O3/2zVP20blxU0M2zWYRKqdFhPnCMDZruHAufSaiEsp0VKIjPEReKtCSodBK0FeLrAp3i4zlz3pSNazQIgQUNlyKKHhpbQS/yOYvC5fLABqKiEpf7X1gDjxz4g6meybKuDpi+PTmfK0kcAlRKB2euSpwLVaguSiiamAgkMrHRxbXUAHnWt2tydsgs2ciGawIH/G3AI9BWi4La6K3upfjJ6B4VcdoDVZIeCX4WgRnXe1QoFWIK/vQS+DBCSNBoqQUOkDPt/eGopA2f/ceuQdxg7ZHQLVgX5Wmt7ImnKCzLcEOj/PCVpVA68nFLEcH81nboQQ+NQGdDTuKSSjKs+1JSsqDkNboR+T2n8iNj45KSP3oy7GjuC1THS2CcBoiaduypFxaLEcbhTMORiJljs1kp6Swnf3ctBSn5UBDMlFdO5WUYAiirwnKjhcyNzZ6YQLtF6ukoj+vwNRcYlcHI5gQqx71bh27GrSRMEQjdE2MycVLWVIEkp1x/A1dHj5/s4/YK2UENZm6yyfMlmzyuhlljksDpfVYkvmsXYA+j+06SBXESqd57z8HBVxFKj8+OW5sK+/prjXp29vabAU0KPubHdRPpCSUotbbKern8ouHRtkaZxPnksAGXMIhiHavwaQW/XnbizU0HmdIQzmEuO08hcZXW6AT2tPWWZTdwfo/l/kzGfyzEvsE6fQap2d35+U7lx4R7XU4y3euSSRD4ZWLLECsSawqZ31kT3iKlWzDSm8rNsL7dUTK1KXI8JJkVyR/M/oEWlv2zXotz0ZvyXnrDh1l51nrC0G48fdgDV7RT4QFD8skwGN6yhKHeBiDaDwdYnRhmueFNGiG97Jq/NhAzI2r8mtUy31tosKCSFtdX44vxxdcaLcW4w+YfYi74jGEiq88ntjg0dkiO98JF1WJB/OrITtNjDQZTy7bEB2PhiJQjAPfOQZswOMRj/egp6k+LbU9s2Pp7akcpEgoNguzwQZoZX6EvdoXIdX9pgf7d7knb+TqliJuzrMBYITQWma3vwCXlFn8

### Proposed solution
Unfortunately, i have none.
It is possible to reach the drop down with keyboard TAB and change it

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/issues?q=is%3Aissue+is%3Aopen+select+label%3Aa11y
